pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -156,15 +156,15 @@ public static String jsonPathString(Object object, String jsonPath, String defau
   /**
    * Extract from Json with path to Long
    */
-  @ScalarFunction
+  @ScalarFunction(nullableParameters = true)
```

Comment:
Let's not annotate this function. When the input is null, we want to return null so that the actual default value defined in the schema can be filled